pep-0723.rst

```diff
@@ -284,6 +319,24 @@ Tools that support managing different versions of Python should attempt to use
 the highest available version of Python that is compatible with the script's
 ``requires-python`` metadata, if defined.
 
+For projects that have large multi-line external metadata to embed like a
+README file, it is recommended that they become directories with a
+``pyproject.toml`` file. If the content is small, for example in the case of
```

Comment:
More explicitly, we could say that whilst this PEP technically allows this use-case, it is strongly discouraged and indicative of the fact that you are now working with a "project" rather than a "script"? (I think you could be more opinionated in general, there are some things that people shouldn't be doing with what 723 enables (I'm personally very sceptical of the embedded configuration support, for example -- if you want to bless it I think you should talk about what happens when multiple files in the same directory tree have ``__pyproject__`` defined)).

Reply:
I put more emphasis on the recommendation